kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain-inputs.yaml

```diff
@@ -38,7 +38,7 @@ managedServerNameBase: managed-server
 managedServerPort: 8001
 
 # WebLogic Docker image
-image: store/oracle/weblogic:12.2.1.3
+image: store/oracle/weblogic:19.1.0.0
```

Comment:
We can't do this until 19.1.0.0 is released.  You're tests will just have to customize this value.

Reply:
@rjeberhard 

Unpatched 12.2.1.3 is invalid for any dynamic cluster used in combination with the introspector, but the introspector always needs to run.   This means that since we don't have patched versions of older WL versions, we must have 19.1 for any test that involves a dynamic cluster (not just new tests - all tests).   

So if we fully parameterize the image-name to avoid hard coding 19.1 into github, we’ll still ultimately need to have a ‘19.1’ reference somewhere to get the ball rolling.  I suppose this means we need a non-github approach to setting the value externally in each environment:   e.g. an internal script on Jenkins that sets up an env var, a Wercker env var, and maybe a reference to /net/some-machine/scratch/image-name.txt for local runs.   

Sound about right?

Reply:
We have a Wercker env var now: IMAGE_NAME_WEBLOGIC and IMAGE_TAG_WEBLOGIC.  I'm open to whatever makes sense for local testing.

Reply:
Resolved via 
https://github.com/oracle/weblogic-kubernetes-operator/pull/489/commits/4de0cf306c42f6d40fdbd24f1ea082930b83ec20